### PR TITLE
Feat/port 3767 suppress webhook creation option

### PR DESCRIPTION
**Pull request by matanl490**

### CODE REVIEW

File: gitlab/gitlab_exporter.py [hunk 1]
```diff
@@ -150,10 +157,10 @@ def process_data_from_all_groups_from_gitlab():
             continue
         
         # Create webhook for root groups
-        if(group['id'] in root_groups_ids):
+        if(SKIP_WEBHOOK_CREATION != 'true' and group['id'] in root_groups_ids):
```
Comment (MPTG94) [inline gitlab/gitlab_exporter.py:160]: Don't need parentheses

File: gitlab/gitlab_exporter.py [hunk 2]
```diff
@@ -150,10 +157,10 @@ def process_data_from_all_groups_from_gitlab():
...
             create_webhook(group['id'])
         
-        get_all_projects_from_gitlab(group['id'], group['name'])
+        # get_all_projects_from_gitlab(group['id'], group['name'])
```
Comment (MPTG94) [inline gitlab/gitlab_exporter.py:163]: Should this stay commented out?

**Review by MPTG94 — APPROVED** [review]
Fix comments and merge

File: gitlab/gitlab_exporter.py (hunk 1 above)
Comment (matanl490) [reply gitlab/gitlab_exporter.py:160]: It is a string

**Pull request merged**